src/Integration/MefServices/IHost.cs

```diff
@@ -29,11 +28,6 @@ namespace SonarLint.VisualStudio.Integration
 {
     internal interface IHost : IServiceProvider
     {
-        /// <summary>
-        /// The UI thread dispatcher. Not null.
-        /// </summary>
-        Dispatcher UIDispatcher { get; }
-
```

Comment:
The host didn't do anything with the dispatcher except expose it as a property.
The only consumer left was the `StateManager`.

(there used to be more consumers of the property before all the simplifications we did in v7, so removing the dispatcher would have involved a lot more work)